packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -31,12 +45,15 @@ export interface NodeHttpHandlerOptions {
 }
 
 interface ResolvedNodeHttpHandlerConfig {
+  requestTimeout?: number;
```

Comment:
I may be reading it incorrectly, but it seems like `requestTimeout` always gets supplied a value in `resolveDefaultConfig` since it defaults to `DEFAULT_REQUEST_TIMEOUT. Can this be non-optional?

Reply:
Good spot. Updated it to non-optional.